_data/release_notes_versions.json

```diff
@@ -1200,5 +1200,23 @@
       ],
       "api-docs": "ballerina-api-docs-1.2.38.zip",
       "release-notes": "ballerina-release-notes-1.2.38.md"
+   },
+   {
+      "version":"1.2.39",
+      "release-date":"2023-05-31",
+      "windows-installer":"ballerina-windows-installer-x64-1.2.39.msi",
+      "windows-installer-size":"148mb",
+      "linux-installer":"ballerina-linux-installer-x64-1.2.39.deb",
+      "linux-installer-size":"145mb",
+      "macos-installer":"ballerina-macos-installer-x64-1.2.39.pkg",
+      "macos-installer-size":"164mb",
+      "rpm-installer":"ballerina-linux-installer-x64-1.2.39.rpm",
+      "rpm-installer-size":"167mb",
+      "other-artefacts":[  
+         "ballerina-1.2.39.zip",
+         "ballerina-1.2.39.vsix"
```

Comment:
Are we planning to release VSIX version 1.2.39?

Reply:
I think we need to remove that. We have not released any versions after 1.2.20.

Reply:
This json is generated one, So I think it is ok to have that, However frontend will remove this.
This is the same approach odf 1.2.38

Reply:
@SasinduDilshara remove it.